### ISSUE TITLE
Tabs

### ISSUE THREAD

**Issue by almson** [lensapp/lens#8080]
Currently, when I navigate from one view to another and back I lose things like search, selection, and have to wait for things to reload. There's a concept in lots of other software called "tabs". "Tabs" allow me to keep several views open, switch between them, and not lose state. I believe they were invented by Netscape Navigator 4. Might want to check it out.

**Comment by Christiaanvdl** [comment]
Thank you for reaching out @almson , i have contacted our development team regarding your request !

**Comment by panuhorsmalahti** [comment]
Hi, Lens 2024 will have tabs. You can already try this version using the beta update channel.

**Comment by jakolehm** [comment]
Tabs are now part of the stable release.

**Comment by almson** [comment]
@jakolehm @panuhorsmalahti @Christiaanvdl  Thanks. Ok, first I figured out that I needed to uninstall `openlens` to jump into the bright future of Lens. But now:

- I'm not exactly what these tabs are supposed to do, lol. I want to open a Pod (its status page, not its yaml) or other object as a tab so I can switch back to it. Ctrl+Click doesn't work. I don't see any button that will turn the sidepane into a tab. In fact Ctrl+Click seems to open my browser in many cases, lol.

- You broke the back button! Why the hell is it switching tabs. Before, I would open a pod, then click its stateful set, etc, and then click back to return. Now I can't. What is back even supposed to do?
